Exporter.py

```diff
@@ -42,6 +42,24 @@ def main(argv):
 
 			elif opt == '--maxtweets':
 				tweetCriteria.maxTweets = int(arg)
+
+			elif opt == '--nolinks':
+				tweetCriteria.noLinks = True
+			
+			elif opt == '--lang':
+				tweetCriteria.lang = arg
+
+			elif opt == '--min_likes':
+				tweetCriteria.min_likes = arg
+
+			elif opt == '--min_replies':
+				tweetCriteria.min_replies = arg
```

Comment:
Hi! could you please explain where do you parse the number of replies for a specific tweet?

Reply:
Hi, the number of replies is parsed from the console as other parameters and then is handled on "TweetManager.py".
For example: 
python3 Exporter.py --querysearch "europe"  --lang "en"  --min_replies 10